assets/js/app.coffee

```diff
@@ -16,18 +16,17 @@ class Choosealicense
       selection.removeAllRanges()
       selection.addRange(range)
 
-  # Qtip position attributes for tooltips
-  qtip_position:
-    my: "top center"
-    at: "bottom center"
-    viewport: $(window)
-    adjust:
-      method: 'shift shift'
-  # Annotation rule types as defined in `_config.yml`
-  ruletypes:
-    permissions: "Permission"
-    conditions: "Condition"
-    limitations: "Limitation"
+  tooltipAttributesMapperByRuleType:
+    permissions:
+      heading: 'Permissions'
+      color: 'hint--success'
```

Comment:
It looks like success/info/error are built into hint.css and while the colors basically match what's needed, I'd probably prefer to separately define permission/condition/limitation colors as success/info/error aren't semantically correct, even if it is just code the end user won't see.

Reply:
@mlinksva  - Updated.